test/integration/targets/mongodb_replicaset/aliases

```diff
@@ -0,0 +1,4 @@
+destructive
+shippable/posix/group1
+skip/osx
+skip/freebsd
```

Comment:
Why are the tests being skipped on macOS and FreeBSD?

Reply:
Wanted to keep things simpler for the moment. Setting up a MongoDB cluster on . FreeBSD is a fair bit different to Linux and I'm not entirely sure a port of mongos is available for sharded clusters..